packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -228,12 +228,20 @@ export interface ISummaryConfigurationHeuristics extends ISummaryBaseConfigurati
      * Defines the maximum number of Ops, since the last received Ack, that can be allowed
      * before running the summary with reason maxOps.
      */
-     maxOps: number;
+    maxOps: number;
     /**
      * Defnines the minimum number of Ops, since the last received Ack, that can be allowed
      * before running the last summary.
      */
     minOpsForLastSummaryAttempt: number;
+    /**
+     * System op weight to use in heuristic summarizing
```

Comment:
might be useful to provide some comment about expected range and how it will be used. Otherwise, it's not very clearly how to use it. For example, is using 1.0 & 1.0 for weights same as 2.0 & 2.0 or not? Do they need to add up to 1.0 together? Things like that are useful to mention (and enforce)